calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -67,253 +67,220 @@ Enabling WAF for certain system services may result in an undesired cluster stat
 
-To view WAF events In Kibana:
+In the previous example, we applied WAF to the `frontend` service of the sample application. Here, we are
+applying WAF to a service of your own application.
 
-1. Create the `tigera_secure_ee_waf*` index pattern. For help with index patterns, see [Kibana index pattern](../visibility/kibana.mdx).
-2. Select the `tigera_secure_ee_waf*` index pattern. 
+1. On the Manager UI, click **Threat Defense**, **Web Application Firewall**.
+2. Select the services you want WAF to inspect, and then click **Confirm Selections**.
+    
+   <img
+    src='/img/calico-enterprise/waf-select-services-3.17.png'
+    alt='WAF services'
+    width='600'
+  />
 
-You should see the relevant WAF assessment from your request.
+3. On the **Web Application Firewall** page, you can verify that WAF is enabled for a service by locating the service and checking that the **Status** column says **Enabled**.
 
-### Create global alerts
+4. To make further changes to a service, click **Actions**, and then **Enable** or **Disable**.
 
-To get notifications on WAF events, create Global Alerts.
+You have now applied WAF rule sets to your own services, and note that the traffic that goes through the selected services will be alerted but not blocked by default.
 
-The following example creates a Global Alert for a SQL Injection attack. Specifically, for Rule ID 942100 [Core Rule Set file](https://raw.githubusercontent.com/coreruleset/coreruleset/v3.3/dev/rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf) 
-that will "deny" all traffic instead of "block" traffic.
+#### Set WAF rule to block traffic
+You can change WAF rule so that the traffic is blocked.
 
-```yaml
-apiVersion: projectcalico.org/v3
-kind: GlobalAlert
-metadata:
-  name: waf-new-alert-rule-info
-spec:
-  summary: 'WAF new waf-alert-942100'
-  description: 'Test WAF Global Alert'
-  severity: 1
-  dataSet: waf
-  period: 1m
-  lookback: 1h
-  query: '"rules.id" = "942100"'
-  threshold: 0
-  condition: gt
-```
-Apply the YAML to your cluster using: `kubectl apply -f test-demo-alert.yaml`
+1. Edit the configmap: 
+   ```bash
+   kubectl edit cm -n tigera-operator modsecurity-ruleset
+   ```
+2. Look for `SecRuleEngine DetectionOnly` and change it to `SecRuleEngine On`.
 
-Now if a SQL Injection attack is detected for rule ID 942100, you will see the global alert in Manager UI, Activity, Alerts.
+| Action | Description                                                                                                                                                             | Disruptive? |
+| ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| DetectionOnly | Traffic is not denied nor dropped. {{prodname}}  will log events. | No
+| On     | Denies HTTP traffic. {{prodname}}  will log the event in Elasticsearch. | Yes          |
```

Comment:
I don't think we need to mention Elasticsearch here at all. We can simply say "will create a security event"

Reply:
Fixed.